packages/components/src/components/popup/popup.scss

```
@@ -0,0 +1,71 @@
+.dc-popup {
+    &__header {
+        padding: 0;
+
+        &--content {
+            display: flex;
+            align-items: center;
+        }
+
+        &__icon {
+            position: absolute;
+            left: 86%;
+        }
+
+        &__button {
+            margin: 0 1.6em !important;
+            width: 13rem !important;
+        }
+
+        &--banner {
+            background-color: var(--status-warning);
+            border-radius: 100px;
+            padding: 0.4rem 0.8rem;
+            height: 22px;
+            display: flex;
+            align-items: center;
```

Since these are elements and not modifiers, I'd suggest the following change for better adherence to BEM concept.
Ideally though, I believe `dc-popup-header` would be the Body and `__content` would be the Element in BEM. Same goes for `body` and `footer`.
```suggestion
        &-content {
            display: flex;
            align-items: center;
        }

        &-icon {
            position: absolute;
            left: 86%;
        }

        &-button {
            margin: 0 1.6em !important;
            width: 13rem !important;
        }

        &-banner {
            background-color: var(--status-warning);
            border-radius: 100px;
            padding: 0.4rem 0.8rem;
            height: 22px;
            display: flex;
            align-items: center;
```

---

@nijil-binary As `https://github.com/binary-com/deriv-app/tree/master/docs/Stylesheet` says, we fallow `BEM Two Dashes style`, the one you are mentioning is `React style`.

---

What I'm suggesting is, since `--banner` or `--content` are elements and not boolean modifiers, they need not be separated using double hyphen (`--`).

---

@nijil-binary thanks for the explanation👍